src/Microsoft.Azure.WebJobs.Host.Storage/StorageClientProvider.cs

```diff
@@ -0,0 +1,192 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using Azure.Core;
+using Microsoft.Extensions.Azure;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Host.Storage
+{
+    /// <summary>
+    /// Abstraction to provide storage clients from the connection names.
+    /// This gets the storage account name via the binding attribute's <see cref="IConnectionProvider.Connection"/>
```

Comment:
This IConnectionProvider comment doesn't seem to apply - I don't see any such logic.